scripts/multiplexContours.py

```diff
@@ -584,6 +669,8 @@ def doPolygonsTouch(poly1, poly2):
 
 def plotBestRegion(bestRegions, expCurve):
 
+    fout = ROOT.TFile("usedExpectedContours.root","recreate")
```

Comment:
maybe just wrap this in a 

```
with ROOT.TFile("....") as fout:
  ...
```

then you drop the `fout.Close()`.

Reply:
Sure

Reply:
Doesn't seem to work with ROOT files.

Reply:
you're right, wow. i am thinking of uproot 🤣